i18n/en.json

```diff
@@ -2256,27 +2256,27 @@
   },
   {
     "id": "api.user.verify_email.broken_token.app_error",
-    "translation": ""
+    "translation": "Bad verify email token type."
   },
   {
     "id": "api.web_socket.connect.upgrade.app_error",
     "translation": "Failed to upgrade websocket connection"
   },
   {
     "id": "api.web_socket_router.bad_action.app_error",
-    "translation": ""
+    "translation": "Unknown websocket action."
   },
   {
     "id": "api.web_socket_router.bad_seq.app_error",
-    "translation": ""
+    "translation": "Invalid websocket message sequence."
```

Comment:
`Invalid sequence for WebSocket messages.` <-- when can this occur?

Reply:
When a developer is sending to the websocket a message with the `seq` field with a value <= 0 (or maybe, no value).